Fix fatal error with llvm 16
error: a function declaration without a prototype is deprecated in all versions of C [-Werror,-Wstrict-prototypes]

Index: compose/asc-globals.c
--- compose/asc-globals.c.orig
+++ compose/asc-globals.c
@@ -455,7 +455,7 @@ asc_globals_get_hint_tag_details (const gchar *tag)
  * Returns: (transfer full): A list of valid hint tags. Free with %g_strfreev
  */
 gchar**
-asc_globals_get_hint_tags ()
+asc_globals_get_hint_tags (void)
 {
 	AscGlobalsPrivate *priv = asc_globals_get_priv ();
 	GHashTableIter iter;
